sites/example/site/one/hypervisor.pan

```diff
@@ -0,0 +1,9 @@
+unique template site/one/hypervisor;
+
+# very very basic
+final variable FILESYSTEM_LAYOUT_CONFIG_SITE = 'site/filesystems/burning';
```

Comment:
ah copy/paste issue, nothing to do with opennebula component, I will remove this line